app/javascript/react/components/MainComponent/MainComponent.js

```diff
@@ -0,0 +1,14 @@
+import React, { Fragment } from 'react';
+import Header from '../Header/Header';
+import Tabs from '../Tabs/Tabs';
+
+const MainComponent = () => {
```

Comment:
naming - what is a MainComponent ?

Main what?

is this the User Dashboard where a user lands?
or Stats Input

hmm no this is main - which is actually a LayoutComponent?

Reply:
Usually I put main component to ensure all component is attached to main component. I think layout component make more sense in this case. I have changed it